atom/common/api/lib/shell.js

```diff
@@ -1 +1,17 @@
-module.exports = process.atomBinding('shell');
+'use strict';
+
+const bindings = process.atomBinding('shell');
+
+exports.beep = bindings.beep;
+exports.moveItemToTrash = bindings.moveItemToTrash;
+exports.openItem = bindings.openItem;
+exports.showItemInFolder = bindings.showItemInFolder;
+
+exports.openExternal = (url, options) => {
+  var activate = true;
+  if (options != null && options.activate != null) {
+    activate = !!options.activate;
+  }
+
+  return bindings._openExternal(url, activate);
```

Comment:
I tend to handle the default parameters in C++, the code is not more complex than JavaScript and avoids the extra wrapper.